service/api_logic/news_logic.py

```diff
@@ -18,4 +17,29 @@ def get_news_by_count(count: int, session) -> str:
             })
         except Exception as e:
             print(f"\033[31mError while receiving blob '{news_record.blob_id}': {e}\033[0m")
+
     return json.dumps(all_results, ensure_ascii=False)
+
+
+def get_latest_sport_news(count: int, sport_name, session):
+    sport = session.query(Sport).filter(Sport.sport_name == sport_name).first()
+    if not sport:
+        return {"error": f"Sport '{sport_name}' not found"}
```

Comment:
Create logic for proper exception/error handling. Less hardcode.

You might wanna look into "Custom Exceptions" on Google, or at least create a method to form the body of the error